src/style.css

```diff
@@ -1,3 +1,72 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic';
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.header__nav{
+  margin: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  margin: 0 50px;
+}
+
+.nav__list {
+  padding: 0;
+  list-style: none;
+  display: flex;
+  margin: 0;
+  gap: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  height: 60px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.new:hover {
+  color: #00ACDC;
+}
+
+.is-active {
+  color: #00ACDC;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  position: relative;
+}
```

Comment:
Double-check, if you have same styles in nav__link then remove it from is-active